README.md

```diff
@@ -1,593 +1,96 @@
-
-### Receiving Answers with Pieces.QGPTQuestionAnswer
-When you get a response back from the copilot after asking a question, you may need to type it accordingly to access the appropriate properties. Here would be an example if getting back the first answer message following a question that was sent over: 
-
-```typescript
-const answer: Pieces.QGPTQuestionAnswer | undefined = result.question?.answers.iterable[0];
-```
-
-Read more on the specifics of the Pieces Copilot logic and [endpoints on this copilot specific repo](https://github.com/pieces-app/pieces-copilot-vanilla-typescript-example).
-
-## Connecting your Application
-When Pieces OS is running in the background of your machine, it is communicating with other local applications that use Pieces software, and up until recently only supporting internally built tools.
-
-As each plugin, extension, or application initializes they 'reach out' to Pieces OS and authenticate with the application themselves. There are a number of application formats that we support and provide for each of our applications. ==When developing on Pieces OS, you can use **"LOCAL_DEV"**
-to avoid any issues with other applications.==
-
-### Creating `Application`
-The `application` model describes what application a format or analytics event originated from. This is passed along when initializing your dev environment and creates a connection to Pieces OS.
-
-To create the `Application` object for your project, you will need to make sure that you have the following two things:
-
-1. Create a `tracked_application` json object to hold your formatted data
-2. Output using `console.log()` following your `connect()` method is complete
-
-***We will go over the different usages in each of these steps one at a time to bring up any specificities that may need a deeper explanation.***
-
-#### Using `tracked_application`
-Connecting your application here is as easy as a single **POST** request and can be done via the Response interface of the **Fetch Api**. Remember that you can name this whatever you would like to, just be sure to include the updated variable name in the `options` down below.
-
-> When creating the `tracked_application` item, you will need to use a type that is not available inside the current `npm_deployment`.
->
-> This structure is the same as the tracked_application full example you see here below, and the only difference from the unavailable type `SeededTrackedApplication` and the available type `TrackedApplication` is `id: number`.
-
-First lets take a look at the `tracked_application` object:
-
-```tsx
-const tracked_application = {  
-    name: Pieces.ApplicationNameEnum.OpenSource,  
-    version: '0.0.1',  
-    platform: Pieces.PlatformEnum.Macos,  
-}
+1. Fork this repository by clicking on the <a href="https://github.com/pieces-app/example-typescript/new/master?readme=1#fork-destination-box"><kbd><b>Fork</b></kbd></a> button.
```

Comment:
Should this be added here for now? This would be only necessary when making a contribution right? We dont need to add it for doing local setup necessarily. So this points seems a bit unclear in the local setup.